testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCTest.java

```diff
@@ -19,18 +19,30 @@
 
 import org.bouncycastle.jce.provider.BouncyCastleProvider;
 import org.jboss.logging.Logger;
+import org.keycloak.common.Profile;
 import org.keycloak.common.util.CertificateUtils;
 import org.keycloak.common.util.KeyUtils;
 import org.keycloak.common.util.MultivaluedHashMap;
 import org.keycloak.common.util.PemUtils;
 import org.keycloak.crypto.KeyUse;
 import org.keycloak.crypto.KeyWrapper;
 import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.utils.KeycloakModelUtils;
+import org.keycloak.protocol.oid4vc.OID4VCClientRegistrationProviderFactory;
```

Comment:
It seems that it is not used.

Reply:
Fixed in [42f5903](https://github.com/keycloak/keycloak/pull/27931/commits/42f5903f054b931386c5c249d4f2990c99ed4673)

Reply:
Thank you.